src/NBitcoin/IBitcoinSerializable.cs

```diff
@@ -51,33 +61,48 @@ public static string ToHex(this IBitcoinSerializable serializable, Serialization
             }
         }
 
-        public static void ReadWrite(this IBitcoinSerializable serializable, byte[] bytes, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION)
-        {
-            ReadWrite(serializable, new MemoryStream(bytes), false, version);
-        }
-        public static void FromBytes(this IBitcoinSerializable serializable, byte[] bytes, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION)
-        {
-            serializable.ReadWrite(new BitcoinStream(bytes)
+		public static void ReadWrite(this IBitcoinSerializable serializable, byte[] bytes, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION, 
+            NetworkOptions options = null)
+		{
+			ReadWrite(serializable, new MemoryStream(bytes), false, version, options);
+		}
+		public static void FromBytes(this IBitcoinSerializable serializable, byte[] bytes, ProtocolVersion version = ProtocolVersion.PROTOCOL_VERSION, 
+            NetworkOptions options = null)
+		{
+            var bms = new BitcoinStream(bytes)
```

Comment:
Same here... maybe format this document?

Reply:
Done